core/src/main/java/com/hcl/ecomm/core/services/impl/DeleteCartItemServiceImpl.java

```diff
@@ -0,0 +1,105 @@
+package com.hcl.ecomm.core.services.impl;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+
+import org.apache.http.client.methods.CloseableHttpResponse;
+import org.apache.http.client.methods.HttpDelete;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClients;
+import org.eclipse.jetty.http.HttpStatus;
+import org.json.JSONObject;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.metatype.annotations.Designate;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.hcl.ecomm.core.config.MagentoServiceConfig;
+import com.hcl.ecomm.core.services.DeleteCartItemService;
+import com.hcl.ecomm.core.services.LoginService;
+
+@Component(
+		immediate = true,
+		enabled = true, 
+		service = DeleteCartItemService.class)
+@Designate(ocd = MagentoServiceConfig.class)
```

Comment:
@rakeshkumar-verma  we don't require to declare MagentoServiceConfig for everySeviceImpl s it is making duplicate configurations on felix console